audobject/core/dictionary.py

```diff
@@ -64,16 +66,32 @@ def update(
 
     def values(self) -> typing.ValuesView[typing.Any]:
         r"""Return values."""
-        return self.__dict__.values()
+        return self._dict_wo_special_attributes.values()
+
+    @property
+    def _dict_wo_special_attributes(self):
+        r"""Return self.__dict__ without special attributes"""
+        d = {}
+        for key, value in self.__dict__.items():
+            if key not in [
+                define.BORROWED_ATTRIBUTES,
+                define.CUSTOM_VALUE_RESOLVERS,
+                define.HIDDEN_ATTRIBUTES,
+                define.KEYWORD_ARGUMENTS,
```

Comment:
Does this mean we also solve a bug here, because we also handle `BORROWED_ATTRIBUTES` etc. now here?

Reply:
yes

Reply:
Do we have additional tests that are able to find this bug?